test/integration/case.test.js

```diff
@@ -51,12 +51,12 @@ describe('Case', () => {
       jwtSecret.secret,
     );
 
-    const caseParams = {
-      organization_id: currentOrg.id,
-      state: 'unpublished'
-    };
+    // const caseParams = {
```

Comment:
Should this still be commented out? If so, do we event need it?

Reply:
Blargh...handling.